Sources/TuistSupport/Errors/ErrorHandler.swift

```diff
@@ -12,27 +12,8 @@ public protocol ErrorHandling: AnyObject {
 
 /// The default implementation of the ErrorHandling protocol
 public final class ErrorHandler: ErrorHandling {
-    // MARK: - Attributes
-
-    /// Function to exit the execution of the program.
-    var exiter: (Int32) -> Void
-
-    // MARK: - Init
-
-    /// Default error handler initializer.
-    public convenience init() {
-        self.init(exiter: { exit($0) })
-    }
-
-    /// Default error handler initializer.
-    ///
-    /// - Parameters:
-    ///   - exiter: Closure to exit the execution.
-    init(exiter: @escaping (Int32) -> Void) {
-        self.exiter = exiter
```

Comment:
If exiter is no longer necessary I think we can get rid of it.

Reply:
That's what this change does 🙂